fail to specify network device

```
❯ sudo ./chaosd attack network delay -d wlo1 -l 1s
2022-07-21T14:51:50.527+0800    INFO    chaosd/network.go:78    Set iptables chains     {"request": ""}
2022-07-21T14:51:50.533+0800    INFO    chaosd/network.go:84    handling tc request     {"tcs": "tcs:{netem:{time:1000000}}"}
2022-07-21T14:51:50.534+0800    INFO    chaosd/network.go:187   get interfaces from ip command{"ifaces": ["lo", "enp42s0", "wlo1", "wlp45s0f3u2u1", "tailscale0", "docker0", "br-b710d8160043", "br-086c929bcc6c"]}
2022-07-21T14:51:50.542+0800    INFO    chaosdaemon/tc_server.go:171    add tc  {"tc": "netem:{time:1000000} device:\"eth0\""}
2022-07-21T14:51:50.542+0800    INFO    chaosdaemon/tc_server.go:209    adding netem    {"device": "eth0", "parent": "root", "handle": "handle 1:"}
2022-07-21T14:51:50.545+0800    ERROR   chaosd/network.go:187   error while adding tc 
```

--- a/pkg/core/network.go
+++ b/pkg/core/network.go
@@ -434,6 +434,7 @@ func (n *NetworkCommand) ToTC(ipset string) (*pb.Tc, error) {
 		Protocol:   n.IPProtocol,
 		SourcePort: n.SourcePort,
 		EgressPort: n.EgressPort,
+		Device:     n.Device,
 	}
 
 	var (

